plugins/PrivacyManager/API.php

```diff
@@ -235,51 +229,126 @@ private function formatAvailableColumnsToAnonymize($columns)
     }
 
     /**
+     * Provide tracker file name and whether it's writable
+     */
```

Comment:
We should try to add some more specific type hinting in the doc blocks where possible
```suggestion
     * @return array{0: string, 1: bool}
     */
```